pypxe/tftp.py

```diff
@@ -140,7 +148,6 @@ def listen(self):
             #Resent those that have timed out
             for i in self.ongoing:
                 if self.ongoing[i]['timeout']+5 < time.time() and self.ongoing[i]['retries']:
-                    print self.ongoing[i]['handle'].tell()
```

Comment:
This line can be safely removed, it was left in when I rewrote this section. Good catch.